src/sentiment-analysis.js

```diff
@@ -0,0 +1,13 @@
+/* This file contains the code for analyzing blog posts text to identify the
+ * negative or positve words being used in a post and return a summary of it
+ * along with a score. The file uses a node module called sentiment to implement
+ * the functionality of analyzing text of blogs. The function accepts plain
+ * text as a parameter i.e text containg no HTML tags, and returns a promise
+ * object whcih contains the result.
+*/
+
+module.exports.run = async function (text) {
+  const Sentiment = require('sentiment');
```

Comment:
This line needs to go above: we only want to `require` this once, not every call to `run()`.  The next line, where you create an instance, is what I wanted in here.

Reply:
This is still not fixed.  Move line 10 out of this function, and above like you had it before.